separates nitrile and latex gloves into their own boxes, fills each

gy1ta23 opened the pull request:
🆑Gy1ta23
tweak: latex gloves and nitrile gloves are now separate
/🆑

Now, we all know latex allergies don't exist in 2311. However, even without the commingling and all that being a health risk, it's still not exactly ideal to have half-filled (or just under) latex glove boxes topped off with two nitrile glove sets, especially given the "half" and "disposable" aspects. Not quite all that efficient, either, having the types mixed, nor entirely sensible by any metric I can dream up. This change will fix this, by turning one box of both types into two of one and filling both appropriately. I appreciate that the sterile glove repacking industry might be upset with this body-blow to their core business, but you always need to break a few knees to make a proper change, as the saying goes.

Spookerton commented on maps/antag_spawn/ert/ert_base.dmm:
```diff
@@ -1013,8 +1013,8 @@
-/obj/item/storage/box/gloves,
-/obj/item/storage/box/gloves,
+/obj/item/storage/box/latexgloves,
+/obj/item/storage/box/latexgloves,
```
```suggestion
/obj/item/storage/box/nitrilegloves,
```

Spookerton commented on code/datums/supplypacks/medical.dm:
```diff
@@ -4,7 +4,8 @@
 /singleton/hierarchy/supply_pack/medical/gloves
 	name = "Refills - Sterile gloves"
-	contains = list(/obj/item/storage/box/gloves = 4)
+	contains = list(/obj/item/storage/box/latexgloves = 2,
+					/obj/item/storage/box/nitrilegloves = 2)
```
```suggestion
	contains = list(
		/obj/item/storage/box/latexgloves = 2,
		/obj/item/storage/box/nitrilegloves = 2
	)
```

Spookerton commented on code/datums/supplypacks/medical.dm:
```diff
@@ -265,7 +266,7 @@
 					/obj/item/clothing/head/surgery/green,
 					/obj/item/storage/box/masks,
-					/obj/item/storage/box/gloves)
+					/obj/item/storage/box/latexgloves)
```
```suggestion
					/obj/item/storage/box/latexgloves
	)
```

Spookerton commented on code/datums/supplypacks/medical.dm:
```diff
@@ -279,7 +280,7 @@
 					/obj/item/storage/box/masks,
-					/obj/item/storage/box/gloves,
+					/obj/item/storage/box/latexgloves,
 					/obj/item/pen)
```
```suggestion
					/obj/item/pen
	)
```

Spookerton commented on code/datums/supplypacks/medical.dm:
```diff
@@ -304,7 +305,7 @@
 					/obj/item/clothing/suit/storage/toggle/labcoat/chemist,
 					/obj/item/storage/box/masks,
-					/obj/item/storage/box/gloves)
+					/obj/item/storage/box/latexgloves)
```
```suggestion
					/obj/item/storage/box/latexgloves
	)
```

Spookerton commented on code/datums/supplypacks/medical.dm:
```diff
@@ -319,7 +320,7 @@
 					/obj/item/tank/oxygen = 5,
 					/obj/item/storage/box/masks,
-					/obj/item/storage/box/gloves)
+					/obj/item/storage/box/latexgloves)
```
```suggestion
					/obj/item/storage/box/latexgloves
	)
```

Spookerton commented on code/datums/supplypacks/medical.dm:
```diff
@@ -356,7 +357,7 @@
 					/obj/item/storage/box/masks,
-					/obj/item/storage/box/gloves,
+					/obj/item/storage/box/latexgloves,
 					/obj/item/storage/belt/medical = 3)
```
```suggestion
					/obj/item/storage/belt/medical = 3
	)
```

Spookerton commented:
While both nitrile and latex gloves are durable and made from rubber, nitrile doesn’t contain the same proteins that are found in latex, which is why they’re safer for people with latex allergies.

With that in mind, let’s check out the pros and cons of nitrile vs latex gloves. This will help you to find out exactly how they differ from each other and which one is best for your needs.

## Latex Gloves

Latex gloves are popular because they’re strong, and this makes them a solid choice in industrial and medical fields. 

### Pros
- Latex gloves fit hands like a second skin, which gives them tactile sensitivity and dexterity.
- The above point makes them valuable for use in sensitive and precise applications, such as surgery.
- They are usually lightly powdered, and this makes them easier to put on and take off.
- They are very elastic, and that increases their level of strength.

### Cons
- Since latex gloves are powdered, this can cause allergic reactions in some people.
- Latex in general is an allergen, which can make it dangerous to use when it comes to healthcare settings.
- Due to their color and how they’re like a second skin, latex gloves can make it difficult for you to detect tears and punctures in them.
- Latex gloves do not handle chemicals and solvents very well, offering minimal protection against them.

## Nitrile GLoves

Nitrile is a synthetic rubber compound that’s often used to make nitrile gloves. 

### Pros
- Nitrile gloves are extremely strong and durable, while being resistant to punctures and tears.
- These gloves come in a variety of colors so that it makes it easier to notice if they have tears in them. Their colors can also help to conceal substances that could be on them, such as tattoo ink or blood.
- Nitrile gloves are highly resistant to chemicals and solvents.
- Medical-grade gloves are usually made out of nitrile because this material is resistant to fats and oils in the body.
- Since they don’t contain any latex, nitrile gloves are safe for people who have latex allergies. This is especially beneficial because some allergic reactions to latex can be life-threatening. An example is anaphylaxis.

### Cons 
- Nitrile gloves tend to be a bit more expensive than latex gloves.
- These gloves are not as comfortable as compared to latex gloves. This is something to bear in mind if you need to wear the gloves for long periods of time.
- Nitrile is said to have less tensile strength than other glove materials, such as latex. Tensile strength is the load that a material can tolerate before cracking or breaking. That said, nitrile still offers excellent protection against tears, cuts, and punctures.

## When Should You Use Nitrile Vs Latex Gloves?

Before you choose between nitrile and latex gloves, it’s good to consider what you’ll need them for. Both types of gloves can be used for almost all the same purposes, but there are some exceptions.

Latex gloves can be used for general usage, dishwashing, janitorial tasks, and medical exams. When it comes to food cooking and preparation, it’s worth bearing in mind that latex gloves are not always recommended as a result of their potential to cause latex allergies. In addition, the powder that’s usually used on latex gloves to make them easier to put on and remove can leave behind a residue on food. Since they have excellent tactile sensitivity, they are recommended for use in healthcare settings, such as dentistry and surgery. 

Nitrile gloves can be used for many of the same tasks that latex gloves are used for, such as general usage, dishwashing, janitorial work, and medical exams. However, since they are powder-free and latex-free, they can also be used to cook and prepare food. This gives them an edge over latex gloves because it makes them more versatile.

## What About Vinyl Gloves?

Vinyl gloves usually come up as an alternative to latex because they are also safe to use by people who have latex allergies.

They also have other advantages. These include how they’re easy to put on and comfortable to wear.

This is because they have a looser fit, but that does come with a big drawback - if the fit is too loose, this can interfere with the gloves’ tactile sensitivity and level of protection.

There are other important drawbacks to know about vinyl gloves. They can’t be used in healthcare settings where there’s a risk of one coming into contact with bodily fluids and blood.

They are permeable to viruses in fluids and tend to leak easily. Since they lack the strength and protection that you want from disposable gloves, they are not as effective as latex and nitrile gloves.

This shitpost brought to you by the content of [gloves.com](https://www.gloves.com)

Spookerton commented:
> oh my god gitkraken doesn't let me squash commits with a coauthor

gitkraken is mid at best, use fork

Spookerton commented:
no it doesn't it's begware

gy1ta23 committed:
glove boxes! seperate! wowie!


give the ert boys nitrile gloves (they're special)

Co-authored-by: Spookerton <spkrtn@pm.me>
yeah i figured i didn't do that list right

Co-authored-by: Spookerton <spkrtn@pm.me>
it was like this before i got here

Co-authored-by: Spookerton <spkrtn@pm.me>
i'm out of commit titles

Co-authored-by: Spookerton <spkrtn@pm.me>
Update code/datums/supplypacks/medical.dm

Co-authored-by: Spookerton <spkrtn@pm.me>
Update code/datums/supplypacks/medical.dm

Co-authored-by: Spookerton <spkrtn@pm.me>
Update code/datums/supplypacks/medical.dm

Co-authored-by: Spookerton <spkrtn@pm.me>
